homeassistant/components/synology_dsm/const.py

```diff
@@ -12,7 +12,6 @@
 )
 
 DOMAIN = "synology_dsm"
-PLATFORMS = ["binary_sensor", "sensor"]
```

Comment:
I think you should:
- In `__init__.py`: remove `_async_platforms`
- here : `PLATFORMS = ["camera", "binary_sensor", "sensor"]`
- In `camera.py.async_setup_entry` line 32 : 
```python
if not SynoSurveillanceStation.CAMERA_API_KEY in api.dsm.apis:
    return True;
```


It will avoid `protected-access` pylint on `__init__.py` and be sure to unload the camera platform if we remove it afterwards.